stdlib/public/SDK/Foundation/Decimal.swift

```diff
@@ -24,13 +24,13 @@ extension Decimal {
 
     public static let pi = Decimal(_exponent: -38, _length: 8, _isNegative: 0, _isCompact: 1, _reserved: 0, _mantissa: (0x6623, 0x7d57, 0x16e7, 0xad0d, 0xaf52, 0x4641, 0xdfa7, 0xec58))
 
-    public var exponent: Int { 
```

Comment:
Please revert any changes in whitespace that aren't related to the bug being fixed.